src/output.rs

```diff
@@ -101,6 +112,92 @@ impl OutputStream for VerboseConsoleOutputStream {
     }
 }
 
+/// Struct that handles JSON console output formatting
+#[derive(Debug, Clone)]
+struct JsonConsoleOutputStream {}
+
+impl OutputStream for JsonConsoleOutputStream {
+    fn print_output(&self, panic_calls: &PanicCallsCollection) {
+        let stream = io::stdout();
+        for (i, trace) in panic_calls.calls.iter().enumerate() {
```

Comment:
We are missing an outer Json Array here. Some tools consider the output invalid JSON.

Reply:
I think this should be considered as a separate option.

For example the best way to handle large JSON files, especially when the "root" object is a large array of similar objects, is to render them as a "stream" of JSON objects.  (For example the `jq` tool works like this natively, and there is also an RFC describing this https://tools.ietf.org/html/rfc7464 )

In the end the difference is minimal, and I can provide the patch for this extra version:

`--json` would create that large array;
`--json-sequence` would create the format I've already proposed;

Reply:
Okay, I was not aware of the spec. Does the current implementation indeed implement the spec, which says: JSON-sequence = *(RS JSON-text LF)?

Reply:
Unfortunately no, it doesn't use the record separator character, mainly because I thought having this "in-between" format is "good-enough", especially since works by default with `jq`, and it can easily be changed with a `sed` script into an actual JSON-object or JSON-sequence.

Moreover fully implementing that RFC would require one to lose all the pretty-printing.

Therefore perhaps the option `--json-sequence` should output a format compliant with that RFC, and another one `--json-stream` could retain the current "good-enough" format.

Reply:
I agree with your reasoning. Let's just change to option name and be done with it.

Reply:
Then just call it `--json-stream` which I guess it's the best "description".